pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/ConsumerStats.java

```diff
@@ -53,7 +53,7 @@
     int getUnackedMessages();
 
     /** Number of average messages per entry for the consumer consumed. */
-    int getAvgMessagesPerEntry();
+    float getAvgMessagesPerEntry();
```

Comment:
This is part of open api, changing this directly may cause compatibility issue.

Reply:
Changed to stay return int, add a getter in `ConsumerStatsImpl`
```java
    // return integer for compatibility
    public int getAvgMessagesPerEntry() {
        return (int) avgMessagesPerEntry;
    }
```